website/content/docs/api-clients/api/pagination.mdx

```diff
@@ -0,0 +1,129 @@
+
+Boundary also includes the top-level response `deleted_ids` as part of the refresh operation.
+The `deleted_ids` response contains the IDs of any resources that were deleted since the initial `list` operation.
+Deleted resources are included in the results of the list for 30 days after they are deleted.
+Resources that have been deleted include the suffix `_deleted`.
+
+After 30 days, the `list_token` expires and you must make a new `list` request without the token to regenerate the full list of results.
+
+## Response structure
+
+All `list` endpoints support the following parameters:
+
+- `list_token` (optional) - An opaque token that is returned from the previous `list` response.
+If you do not supply a value, pagination starts from the beginning.
+- `page_size` (optional) - An unsigned integer that indicates the number of items that should be included on the page of search results.
+If you do not include a value, or if you include a value of `0`, Boundary uses the default page size of 1000 items.
+If you configure it, the `page_size` value overrides the default page size.
+Controller administrators can also configure a `max_page_size` option that constrains the maximum page size a user can request.
+
+Boundary returns the following response parameters:
+
+- `response_type` - Defines whether the response is a `delta` or `complete` listing.
+The value is `delta` until the last page is returned, at which point it will be `complete`.
+- `list_token` - An opaque token which is included in subsequent listing requests after the initial one, so that the client can continue to paginate the results.
+- `sort_by` - The column name that the returned items should be sorted on.
+Each list endpoint has one defined column which defines the sorting for all returned items.
+This attribute is included in the response, so that the sort order for each endpoint is explicit.
+- `sort_dir` - The direction in which the returned items are sorted.
+Responses can include `desc` for descending, or `asc` for ascending.
+- `items` - The list of items.
+The items represent replacements for any existing items with matching IDs in the client's cache.
+- `est_item_count` - An unsigned integer that indicates the total number of items that are available in the list of results.
+This number may be an estimate, since the number of items may change during the pagination process.
+
+### Example response
+
+```json
+{
+  "response_type": "<delta|complete>",
+  "pagination_token": "<opaque-token>",
+  "sort_by": "created_time",
+  "sort_dir": "desc",
+  "est_item_count": 1000,
+  "items": [
+    {
+      "id": "ttcp_1234567890",
+      "scope_id": "p_1234567890"
+    },
+    {
+      "id": "ttcp_2234567890",
+      "scope_id": "p_1234567890"
+    }
+  ]
+}
+```
+
+## List tokens
+
+List tokens contain general metadata associated with the ongoing pagination and either a pagination or refresh token, depending on the phase of the pagination.
```

Comment:
This contains internal implementation details that I don't think we need to mention
```suggestion
List tokens contain general metadata associated with the ongoing pagination.
```